### DISCUSSION TITLE
Battery performance on specific Tuf model.

### DISCUSSION THREAD

**Discussion by Billvass** [seerge/g-helper#244] [General]
Hi there! I really like the app and find it very useful since it makes using the system hustle free. Although, it works great I am dealing with very bad igpu performance while the device is unplugged (fx506hc model). Is there any way this issue can be solved on your end? It would help a lot since I have noticed that every other person on reddit with this model is dealing with the same issue, so it has to be a software issue.

**Comment by seerge** [comment]
Hello, your igpu performance is not controlled anyhow specially with the app :) you can try to set higher custom PPTs for your system under fans+power, m.b. that will give it more headroom?

**Reply by Billvass** [reply]
Unfortunately changing the fan curve from the Fans+power section does not change the overall performance. I believe that the solution seems to be on a lower level than the app itself. It is so bad that the Asus engineers do not seem to care to provide a solution.

**Reply by seerge** [reply]
@Billvass i meant not the fans, but power limits? or you don’t see that slider in your case? (if so - then they are not supported on your model :( )

**Reply by Billvass** [reply]
No, there is no such option on my device

**Reply by seerge** [reply]
Oke, then i’m afraid even that setting is not available:(

**Reply by Billvass** [reply]
Anyway, thank you for your time.
